rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -26,63 +26,72 @@ Create and document the environment for the automation of processes.
 
-Global variables and credentials are used for values that are the same in multiple processes and which frequently change. This way you can reduce the time involved and the error rate in the maintenance of your processes. You can link global variables and credentials anywhere that a globe symbol is displayed.
+You use global variables and credentials for values that are the same in multiple processes and which frequently change. This way you can reduce the time involved and the error rate in the maintenance of your processes. You can link global variables and credentials anywhere that a globe symbol is displayed.
 
 === Manage Categories
 
-Projects are assigned to categories. A category describes the environment of a process in a few keywords.
+Project managers organize projects into categories. A category describes the environment of a process in a few keywords, for example `Customer Support Mystical University` or `Finance`.
 
-A project can only be assigned to a single category. First create a standard category, so you can start with process automation quickly. You can change the category of a process later. After you have automated the first projects, it makes sense to create an archive category and move completed projects there.
+A project can only belong to a single category. First, create a standard category, so you can start with process automation quickly. You can change the category of a process later. After you have automated the first projects, create an archive category and move completed projects there.
 
 ==== Create a Category
 
-Create categories to group processes. Assign a process to a category during evaluation or project creation.
+Create categories to group processes.
+
+Assign a process to a category during evaluation or project creation.
 
-. In the *Process Management* view of the *Process Automation* module, click *Create*.
+. Open the *Process Management* view of the *Process Automation* module.
+. Click *Create*.
 . Enter a name in the *Create a category* window.
 . Click *OK*.
 
 ==== Edit a Category
 
 Edit a category to render its name more precisely.
 
-. In the *Process Management* view of the *Process Automation* module, click the *Edit* icon image:edit-icon.png[pen-to-square symbol,1.5%,1.5%] on the panel of the category you want to edit.
+. Open the *Process Management* view of the *Process Automation* module.
+. Click the *Edit* icon image:edit-icon.png[pen-to-square symbol,1.5%,1.5%] on the panel of the category you want to edit.
 . Change the name in the *Edit the category* window.
 . Click *OK*.
 
 ==== Remove a Category
 
 Remove categories you no longer need.
 
-Only categories that do not contain processes can be deleted.
+You can only remove categories that do not contain processes.
```

Comment:
I would add a lead-in to the procedure.

Reply:
used the second sentence as lead.in with colon